opentasks-provider/src/main/java/org/dmfs/provider/tasks/TaskProvider.java

```diff
@@ -897,6 +928,17 @@ public Uri insertInTransaction(final SQLiteDatabase db, Uri uri, final ContentVa
 
                 break;
 
+            case INSTANCES:
+            {
+                InstanceAdapter instance = mInstanceProcessorChain.insert(db, new ContentValuesInstanceAdapter(values), isSyncAdapter);
+                rowId = instance.id();
+                result_uri = TaskContract.Tasks.getContentUri(mAuthority);
```

Comment:
Shouldn't this be `TaskContract.Instances.getContentUri(..)`? Isn't `rowId` the id of the instance, not the task here?

Reply:
good catch, fixed it.